ush/forecast_predet.sh

```diff
@@ -241,13 +241,19 @@ FV3_GFS_predet(){
     rprefix=enkf$rCDUMP
     memchar=mem$(printf %03i $MEMBER)
   fi
-  memdir=$ROTDIR/${prefix}.$PDY/$cyc/atmos/$memchar
+  memdir=${memdir:-$ROTDIR/${prefix}.$PDY/$cyc/atmos/$memchar}
   if [ ! -d $memdir ]; then mkdir -p $memdir; fi
+  if [[ $RUN == "gefs" ]]; then
+    if [ ! -d ${memdir}/history ]; then mkdir -p ${memdir}/history; fi
```

Comment:
With `mkdir -p`, you shouldn't need to check if the directory already exists first.

Reply:
I added the check to avoid run mkdir if it is already exists. Of course, it doesn't matter for "mkdir -p". I can delete the check if you want.